ext/bcmath/libbcmath/src/sub.c

```diff
@@ -33,6 +33,7 @@
 #include "private.h"
 #include <stddef.h>
 #include <stdbool.h>
+#include <string.h>
```

Comment:
If this is for ``size_t`` (and only it) then this is covered by ``stddef.h``

Reply:
This is also needed for `memset()`